src/Attributes/NoArgumentsBase.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Consolidation\AnnotatedCommand\Attributes;
+
+use Consolidation\AnnotatedCommand\Attributes\AttributeInterface;
+use Consolidation\AnnotatedCommand\Parser\CommandInfo;
+
+abstract class NoArgumentsBase
```

Comment:
OK this seems generally useful, but I don't understand why it's called NoArgumentsBase. Either move to Drush, or include a comment that shows how and when to use it here.

Reply:
Moved to Drush